apps/studio/components/interfaces/Organization/BillingSettings/Subscription/SubscriptionPlanUpdateDialog.tsx

```diff
@@ -568,6 +572,16 @@ const SubscriptionPlanUpdateDialog = ({
                     )}
                 </div>
               )}
+
+              {projects.length === 0 && (
+                <div className="pb-2">
```

Comment:
<img width="807" alt="Screenshot 2025-04-11 at 22 42 27" src="https://github.com/user-attachments/assets/e7e3f95d-bcbc-4257-9b18-58a660981c7d" />

Maybe we can reduce the blank space underneath the estimation a bit? Seems like this more in space is caused by this change.

Reply:
I don't think it's caused by this change, but rather because the right-side is so long and it just spaces it out. We're going to add billing address in a separate PR too (collaposed to one/two lines) that would fill the gap a bit

Reply:
oh yeah, you're right 👍